src/plugins/test/accounting_test01.cpp

```diff
@@ -73,10 +72,10 @@ void initialize_map (
 {
     Association user1 = {"bank_A", 0.5, 5, 0, 7, 0, {},
                          {}, 0, 1, {"*"}, "*", 2147483647, 2147483647, 0, 0,
-                         {}, {}};
+                         {}};
     Association user2 = {"bank_A", 0.5, 5, 0, 7, 0, {},
                          {}, 0, 1, {"*"}, "*", 2147483647, 2147483647, 0, 0,
-                         {}, {}};
+                         {}};
 
```

Comment:
Are these whitespace changes in the right commit?

Reply:
Those changes are actually removing an empty map from each `Association` object because its corresponding member in the Association object (`queue_held_jobs`) was also removed in this commit.

Reply:
Oh, duh, I see that now. I misread the diff :facepalm: